data/test.schema.json

```diff
@@ -100,6 +101,27 @@
           "prefix"
         ]
       }
+    },
+   "stdout": {
```

Comment:
it looks like there's an extra indent at this, level. Other than that the schema update looks good. Thanks!